### PR TITLE
Only track XB views when scrolled into view

### LINKED ISSUE #75: XB Analytics: To add a way to collect views on XBs on a webpage for further analysis
From [XB Analytics Discussion](https://github.com/humanmade/platform-dev/issues/533#issuecomment-708541570) in Platform Dev repo

Currently we track all page views and XB views when the page loads, but XB block tracking is not influenced by being visible in the viewport.

### User Story
As a marketer / publisher (user of XBs), I want to be able to track the no. of views on each of my XB variants on a webpage so that I have an understanding on how each of the XB variant performed compared to rest of the page. 

### Proposed Tactic
We should be able to measure the view on a XB by counting one when they are scrolled into view. 
Thereafter the no. can be used to compare with overall page views

### Acceptance Criteria
- [ ] Given an XB on a webpage, When the XB comes into view, Then we track a view on that XB
- [ ] Given an XB is viewed more than once for the same page, When the XB comes into view, Then we only track the first view

### PR DESCRIPTION
**Pull request by roborourke**
Fixes https://github.com/humanmade/altis-analytics/issues/75

### CODE REVIEW

File: src/experiments.js [hunk 1]
```diff
@@ -1,3 +1,17 @@
+/**
+ * Utils.
```
Comment (jazzsequence) [inline src/experiments.js]: We should have more descriptive docblocks here. Not only does this not explain what the function does, it's potentially misleading.

**Review by jazzsequence — CHANGES_REQUESTED** [review]
Minor request to expand the docblock for the `isVisible` function. It's fairly self-explanatory in the code itself, but the comment `Utils.` by itself is confusing and does not help the person reading the code.

File: src/experiments.js (hunk 1 above)
Comment (roborourke) [reply src/experiments.js]: Oh yeah, that docblock was for going to be for a section of the file rather than the function. I'll update now

**Review by jazzsequence — APPROVED** [review]
🤘

**Pull request merged**